Build of test unit script_P2SH fails

CXX      test/test_test_navcoin-script_P2SH_tests.o
test/script_P2SH_tests.cpp: In member function ‘void script_P2SH_tests::sign::test_method()’:
test/script_P2SH_tests.cpp:115:121: error: no matching function for call to ‘CScriptCheck::CScriptCheck(CCoins, CMutableTransaction&, int, int, bool)’
             bool sigOK = CScriptCheck(CCoins(txFrom, 0), txTo[i], 0, SCRIPT_VERIFY_P2SH | SCRIPT_VERIFY_STRICTENC, false)();
                                                                                                                         ^
In file included from test/script_P2SH_tests.cpp:8:0:
./main.h:439:5: note: candidate: CScriptCheck::CScriptCheck(const CCoins&, const CTransaction&, unsigned int, unsigned int, bool, PrecomputedTransactionData*)
     CScriptCheck(const CCoins& txFromIn, const CTransaction& txToIn, unsigned int nInIn, unsigned int nFlagsIn, bool cacheIn, PrecomputedTransactionData* txdataIn) :
     ^~~~~~~~~~~~
./main.h:439:5: note:   candidate expects 6 arguments, 5 provided
./main.h:438:5: note: candidate: CScriptCheck::CScriptCheck()
     CScriptCheck(): amount(0), ptxTo(0), nIn(0), nFlags(0), cacheStore(false), error(SCRIPT_ERR_UNKNOWN_ERROR) {}
     ^~~~~~~~~~~~
./main.h:438:5: note:   candidate expects 0 arguments, 5 provided
./main.h:425:7: note: candidate: CScriptCheck::CScriptCheck(const CScriptCheck&)
 class CScriptCheck
       ^~~~~~~~~~~~
./main.h:425:7: note:   candidate expects 1 argument, 5 provided
./main.h:425:7: note: candidate: CScriptCheck::CScriptCheck(CScriptCheck&&)
./main.h:425:7: note:   candidate expects 1 argument, 5 provided
Makefile:8723: recipe for target 'test/test_test_navcoin-script_P2SH_tests.o' failed
make[2]: *** [test/test_test_navcoin-script_P2SH_tests.o] Error 1
make[2]: Leaving directory '/home/alex/navcoin-core/src'
Makefile:9155: recipe for target 'all-recursive' failed
make[1]: *** [all-recursive] Error 1
make[1]: Leaving directory '/home/alex/navcoin-core/src'
Makefile:683: recipe for target 'all-recursive' failed
make: *** [all-recursive] Error 1

--- a/src/test/script_P2SH_tests.cpp
+++ b/src/test/script_P2SH_tests.cpp
@@ -107,12 +107,14 @@ BOOST_AUTO_TEST_CASE(sign)
     }
     // All of the above should be OK, and the txTos have valid signatures
     // Check to make sure signature verification fails if we use the wrong ScriptSig:
+
     for (int i = 0; i < 8; i++)
         for (int j = 0; j < 8; j++)
         {
+            PrecomputedTransactionData txdata(txTo[i]);
             CScript sigSave = txTo[i].vin[0].scriptSig;
             txTo[i].vin[0].scriptSig = txTo[j].vin[0].scriptSig;
-            bool sigOK = CScriptCheck(CCoins(txFrom, 0), txTo[i], 0, SCRIPT_VERIFY_P2SH | SCRIPT_VERIFY_STRICTENC, false)();
+            bool sigOK = CScriptCheck(CCoins(txFrom, 0), txTo[i], 0, SCRIPT_VERIFY_P2SH | SCRIPT_VERIFY_STRICTENC, false, &txdata)();
             if (i == j)
                 BOOST_CHECK_MESSAGE(sigOK, strprintf("VerifySignature %d %d", i, j));
             else

